zk/debug_tools/nightly-block-compare/main.go

```diff
@@ -250,7 +250,8 @@ func main() {
 	}
 
 	if zkevmLatestBlock != nil && new(big.Int).Abs(new(big.Int).Sub(zkevmLatestBlock, sequencerLatestBlock)).Cmp(big.NewInt(int64(*blockHeightDiff))) > 0 {
-		log.Printf("Warning: zkEVM node is more than %d blocks apart from Sequencer: zkEVM at %d, Sequencer at %d", *blockHeightDiff, zkevmLatestBlock, sequencerLatestBlock)
+		log.Printf("Error: zkEVM node is more than %d blocks apart from Sequencer: zkEVM at %d, Sequencer at %d", *blockHeightDiff, zkevmLatestBlock, sequencerLatestBlock)
+		os.Exit(1)
```

Comment:
We still want comparison to happen though. So we just need to remember that this test fails and return the right exit code at the end.

Reply:
ahh - yeah no worries